src/Workloads/Workloads.Autorest/docs/New-AzWorkloadsSapVirtualInstance.md

```diff
@@ -12,22 +12,23 @@ Creates a Virtual Instance for SAP solutions (VIS) resource
 
 ## SYNTAX
 
-### CreateWithDiscovery (Default)
+### Register an existing SAP system as a VIS
```

Comment:
Could you explain why this updated? Generally there should be the name of parameter set.

Reply:
Thought this is a heading for the document, I reverted back to parameter set. Can you please check now?